dist/api_data.py

```diff
@@ -1263,7 +1263,7 @@ def __ge__(self, other):
         available, and "print" selects a format where only non-printing
         characters are hexadecimal encoded.  These formats are compatible
         with the @ref util_dump and @ref util_load commands''',
-        choices=['hex', 'json', 'print']),
+        choices=['hex', 'json', 'pretty', 'print']),
```

Comment:
The paragraph above the change explains what the options do, so `pretty` needs some wording.

I don't see how `pretty` would be compatible with `load`, so that will need to be spelled out. I wouldn't go all the way to "debugging only", but it's really only useful for humans that want to read stuff.

And, the Reference Guide documentation for the command-line utility has to change.

Reply:
55a5ee4